src/main/kotlin/com/github/biomejs/intellijbiome/GeneralProcessCommandBuilder.kt

```diff
@@ -48,9 +50,34 @@ class GeneralProcessCommandBuilder : ProcessCommandBuilder {
         command.withExePath(exec)
         workingDir?.let { command.withWorkingDirectory(Path(it)) }
         inputFile?.let { command.withInput(File(it.path)) }
-        command.addParameters(parameters)
         charset?.let { command.withCharset(it) }
 
-        return BiomeTargetRun.General(command)
+        // patch the command line to be run on WSL 2 instead of the host
+        val wslDistribution = WslPath.getDistributionByWindowsUncPath(exec)
```

Comment:
Can you help me to understand the logic behind it?

Reply:
At this point, `exec` is like `\\wsl.localhost\Ubuntu\path\to\biome`. This is called an [UNC path](https://learn.microsoft.com/en-us/dotnet/standard/io/file-path-formats#unc-paths), and `Ubuntu` part points a distribution in WSL. We can have multiple distributions in WSL, so we need to know which the Biome executable is installed in.

Also, this can be used for checking whether the path is in WSL or not. `WslPath.getDistributionByWindowsUncPath` returns `null` when the path is not an UNC path of WSL.

Reply:
Thank you for your explanation!
I'm just wondering if EelApi can help us here:

https://github.com/JetBrains/intellij-community/blob/bb7efcadbabbd595a3c0e9db5f0e49e786f59834/platform/eel/src/com/intellij/platform/eel/EelApi.kt#L9

Reply:
@denbezrukov In my understanding, the EEL API is lower layer, and it is used inside WSL integration: https://github.com/JetBrains/intellij-community/blob/084dd2dec1f636114a7779bb2eb82c569b4d6054/platform/platform-impl/src/com/intellij/execution/wsl/WslIjentUtil.kt#L139 and https://github.com/JetBrains/intellij-community/blob/084dd2dec1f636114a7779bb2eb82c569b4d6054/platform/platform-impl/src/com/intellij/execution/wsl/WSLDistribution.java#L248. Thus I think it still makes sense to use WSLDistibution as higher layer implementation.